index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Ishpaul Singh</title>
+</head>
+<body>
+    <header>
+        <!--The nav bar-->
+        <nav>
+            <!--logo-->
+            <div class="logo">
+                <h1>ISHPAUL SINGH</h1>
+            </div>
+            <!--menu icon-->
+            <div class="toggle-menu">
+                <a href="#"><img src="assets/icons/menu-icon.svg" alt="menu"></a>
+            </div>
+        </nav>
+
+    </header>
+        <!--INTRODUCTION SECTION-->
+        <section class="intro-section">
+            <!--HEADLINE-->
+            <h1 class="headline">
+                Hii!<br>
+                I'm Ishpaul Singh<br>
+                Glad to see you!
+            </h1>
+            <!--description-->
+            <p class="description">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <!--social-handles-->
+            <div class="social-handles">
+                <p>Let's Connect</p>
+                <ul class="social-icons">
+                    <li class="github"><a href="#"><img src="/assets/icons/github.svg" alt="github"></a></li>
+                    <li class="linkedin"><a href="#"><img src="/assets/icons/linkedin.svg" alt="linkedin"></a></li>
+                    <li class="twitter"><a href="#"><img src="/assets/icons/twitter.svg" alt="twitter"></a></li>
+                    <li class="instagram"><a href="#"><img src="/assets/icons/instagram.svg" alt="instagram"></a></li>
+                </ul>
+            </div>
+        </section>
+
```

Comment:
- [ ] Use the font family , space, size similar or closer to the provided design 
This is your site 
![image](https://user-images.githubusercontent.com/43877689/170300806-d7e93e6a-bae5-4a8b-adab-b22dde650a8a.png)

This is Microverse Design 
![image](https://user-images.githubusercontent.com/43877689/170301127-0c3eab2b-fb00-4d74-a141-a0957cd4e676.png)

Ignore the red rectangle. But my suggestion is to consider the font used, the size, font weight, etc.